app/routes/explore.js

```diff
@@ -249,4 +249,9 @@ export default class ExploreRoute extends Route {
   queryParamsDidChange(change, params) {
     this.debouncedFilterChange(params);
   }
+
+  resetController(controller) {
+    super.resetController(...arguments);
+    controller.clearAllFilters();
```

Comment:
Why not using clearFilter(), this results in duplication of code

Reply:
clear filter clears only one filter at a time. at resetController i need to call it for every filterOptions.